app/controllers/api/service_templates_controller.rb

```diff
@@ -20,6 +20,14 @@ def create_resource(_type, _id, data)
 
     def edit_resource(type, id, data)
       catalog_item = resource_search(id, type, collection_class(:service_templates))
+      if data.key?("schedule_time")
+        schedule_time = data&.delete("schedule_time")
+        if schedule_time.nil?
```

Comment:
Do we have a way to make the delete more explicit?  It's a little hidden that passing `schedule_times => nil` in the hash performs a deletion.  I'm good with this as a first pass if we can't do that easily.

Reply:
@jrafanie Since this request is on the service template we can't really do it with a `delete` action. We could make it more explicit by requiring the call to pass "delete" as the value of the schedule_time in the `edit` action request. What do you think about that?

Reply:
I think that makes more sense.  We're pretty stuck because the delete action isn't what we want to do.  I'm fine with doing this using the string "delete" or "remove"

Reply:
Is that how we work with CRUD actions on other subcollections?